apps/src/templates/teacherDashboard/SectionSelectionDropdown.story.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import SectionSelectionDropdown from './SectionSelectionDropdown';
+
+const sections = [
+  {
+    name: 'Assigned Section',
+    id: 1,
+    isAssigned: true
+  },
+  {
+    name: 'Unassigned Section',
+    id: 2,
+    isAssigned: false
+  },
+  {
+    name: 'Selected Section',
+    id: 3,
+    isAssigned: false
+  }
+];
+
+export default storybook => {
+  storybook.storiesOf('SectionSelectionDropdown', module).addStoryTable([
+    {
+      name: 'SectionSelectionDropdown',
+      story: () => (
+        <div>
+          <SectionSelectionDropdown
+            selectedSection={sections[2]}
+            sections={sections}
+          />
```

Comment:
please add the selectSection prop like this example, so you can see when it gets called: https://github.com/code-dot-org/code-dot-org/blob/1cad5ac667f9ed1a3aa31063e6d06bd21a78e2bd/apps/src/templates/projects/publishDialog/PublishDialog.story.jsx#L19